Pod/Classes/NSURLSession/NSURLSession+Mock.swift

```diff
@@ -23,6 +23,13 @@ public enum RequestDebugLevel: Int {
 }
 
 /**
+ An opaque reference to a request for mocking - this can be passed back in to determine the state of a mock
+*/
+public struct Handle {
```

Comment:
It can't because Swift doesn't support that :( It has to be a concrete implementation, not an abstract protocol. And all my playing around to make it work hits the wall of `Equatable` being a bit rubbish.